pump/server.go

```diff
@@ -376,7 +376,12 @@ func (s *Server) Start() error {
 	// sets a timeout for the read of matchers
 	m.SetReadTimeout(time.Second * 10)
 
-	grpcL := m.Match(cmux.HTTP2HeaderField("content-type", "application/grpc"))
+	// grpcL := m.Match(cmux.HTTP2HeaderField("content-type", "application/grpc"))
```

Comment:
Why do we keep this line?

Reply:
I think we can and better just delete it.